exec/java-exec/src/test/java/org/apache/drill/exec/fn/impl/TestAggregateFunctions.java

```diff
@@ -460,4 +466,23 @@ public void testGroupBySystemFuncFileSystemTable() throws Exception {
   public void test4443() throws Exception {
     test("SELECT MIN(columns[1]) FROM dfs_test.`%s/agg/4443.csv` GROUP BY columns[0]", TEST_RES_PATH);
   }
+
+  @Test
+  public void testCountStarRequired() throws Exception {
+    final String query = "select count(*) as col from cp.`tpch/region.parquet` limit 0";
```

Comment:
Only schema is being validated in this case.
It might be confusing if the query returns the records but we only inspect the schema.